src/App.test.tsx

```diff
@@ -1,9 +1,3 @@
-import React from 'react';
-import { render } from '@testing-library/react';
-import App from './App';
-
-test('renders learn react link', () => {
-  const { getByText } = render(<App />);
-  const linkElement = getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
-});
+import React from "react";
+import { render } from "@testing-library/react";
+import App from "./App";
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
'App' is defined but never used. [@typescript-eslint/no-unused-vars](https://eslint.org/docs/rules/@typescript-eslint/no-unused-vars)